### ISSUE TITLE
Manim Notebook Logo Uplift

### ISSUE THREAD

**Issue by Splines** [Manim-Notebook/manim-notebook#100]
### Current Logo

I love the idea that our Logo is created using Manim itself! However, I also see the shortcomings that come with Manim not being a design tool like Figma or Adobe Illustrator.

<img width="248" alt="logo" src="https://github.com/user-attachments/assets/e0803c6b-c7c8-42d3-944f-1f4b77d0b538" />

What I like about the current logo:
- Pick ups the "M" in LaTeX font
- has a notebook in it
- is simplistic
- is created with Manim itself!

What I dislike about the current logo:
- White shine is cut-off at border
- Logo is not in square format (1:1)
- While the logo is simplistic, in the center there's just too much going on with many different lines crossing since the notebook itself has no fill
- Only strokes are used such that no area has a bit more fill to stand out.

### New Logo proposal

I'm not at all an experienced designer, so this is the best I can come up with 😅 What do you think about it @bhoov? I do have it as SVG available.

I tried to:
- Have the theming of the "Notebook" prominent by recreating a good-old spiral book
- Give a teaser of Manim Cells via stylized "code blocks"
- Have a catchy background to stick out in the VSCode Marketplace
- Have it in format 1:1 and go for a circle as cut-off mask as is common on the Marketplace as well as on multiple social media platforms

Here the logo proposal as well as how it could look like in the VSCode extension tab.

<img src="https://github.com/user-attachments/assets/e566c84b-2a21-47ea-8983-4f18729d9ecb" width="300px"/>

<img src="https://github.com/user-attachments/assets/9f26e8a0-4400-438b-b12a-27508bc8a695" width="250px"/>

**Comment by Splines** [comment]
Thanks for the feedback, I will work on it based on your suggestions and make a new proposal.

**Comment by Splines** [comment]
@bhoov What do you think about more simple version that emphasizes the _M_:

<img src="https://github.com/user-attachments/assets/146ea64e-c0f0-454f-94a4-0ae26fdf86b0" width="300px">
<img src="https://github.com/user-attachments/assets/0ca4c7f4-877b-4538-bc2e-4a62b1c61a38" width="300px">

Then maybe for a header:

![manim-notebook-headerlo](https://github.com/user-attachments/assets/e96d535d-84b8-437a-99dc-50b6e51c7e28)

**Comment by bhoov** [comment]
I do like it more! Perhaps the M could be a bit "bolder" so perhaps it stands out more in the small logo size? And do you think there is a way to put a "notebook" association into the logo in an elegant way? No stress if not, we can definitely run with the current version.

**Comment by Splines** [comment]
> Perhaps the M could be a bit "bolder" so perhaps it stands out more in the small logo size?

You're right that the M could be a bit bolder. Unfortunately, the standard LaTeX `\textbf{M}` looks kind of ugly and makes it a lot wider:

<img src="https://github.com/user-attachments/assets/f52056c9-813a-4cb3-abf0-d8804b13a741" width="180px">


That's why compared to the previous version, I've removed the inner stroke to have a bigger white or reddish area, then I additonally added a kind of drop shadow to further increase the fill area. How do you like that one?

<img src="https://github.com/user-attachments/assets/bf2eb0ee-f864-4c5a-9c60-02fd90c94e5d" width="300px">

---

![manim-notebook-header-bolder](https://github.com/user-attachments/assets/1350afcf-27e1-4b7d-9540-9191f643e95e)

<img src="https://github.com/user-attachments/assets/f1e5b7c8-72e9-478d-8103-996a9a53168e" width="250px">

---

> And do you think there is a way to put a "notebook" association into the logo in an elegant way? No stress if not, we can definitely run with the current version.

I wish I knew an elegant way to do so but I didn't find one... Note that I have a [Noun Project](https://thenounproject.com/) subscription (for the icons, and they can be used comercially), so if you find a nice notebook logo there, that could also be a way.

But the logo got too crowded all the time I tried to incorporate a notebook, which is why I went to the more simple version. E.g. this one is inspired by the placement of the shapes in the [Manim Community logo](https://www.manim.community/logo.svg), but I had to scale down the M, the whole Logo doesn't fit into a circle anymore and I don't like it too much...

<img src="https://github.com/user-attachments/assets/bdba7125-d319-48e0-a821-261cf903cabc" width="250px">

**Comment by bhoov** [comment]
I see. Between the two, I say let's keep your [original notebook logo](https://private-user-images.githubusercontent.com/37160523/395553210-e566c84b-2a21-47ea-8983-4f18729d9ecb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2NjI3NjgsIm5iZiI6MTczNTY2MjQ2OCwicGF0aCI6Ii8zNzE2MDUyMy8zOTU1NTMyMTAtZTU2NmM4NGItMmEyMS00N2VhLTg5ODMtNGYxODcyOWQ5ZWNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDE2Mjc0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhNWRjM2ZhNDBlNjdmZWMxNTk1NmE2YTkwY2NhZWIyNmZkYjk3YTQ3MTVlM2JlMjQ1MTRmZjU0MDBjZjQ3NjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2N5V1sz16rtJ4Tuf_V7Yim63c0IRDdskf-zDHNK_81o) with the displayed cells. From your attached [screenshot](https://private-user-images.githubusercontent.com/37160523/398612106-f1e5b7c8-72e9-478d-8103-996a9a53168e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2NjI1OTYsIm5iZiI6MTczNTY2MjI5NiwicGF0aCI6Ii8zNzE2MDUyMy8zOTg2MTIxMDYtZjFlNWI3YzgtNzJlOS00NzhkLTgxMDMtOTk2YTlhNTMxNjhlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDE2MjQ1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4MjNhMWE1YWY1MWM1MWE4MmUzNmVkZDRlYTUwMTdhNjc5YzFjMGFlMWQzYTY2NjllNzE4MTIxMjBhNmIxNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PuuXlAezeO5yhwbz2_ZIii8Wg7SxTdv477WATOnEuUM), it looks like other extensions have the simple latex `M` as their logo (e.g., `math-snippets`). 

What do u think?

**Comment by Splines** [comment]
When you link to the images, I cannot open them with the error "This private-user-images.githubusercontent.com page can’t be found".

As a combination of the two logos, what do you think about this one?

![Manim Notebook Logob](https://github.com/user-attachments/assets/55ea387a-66ad-4447-9154-9c82c50c1316)

**Comment by Splines** [comment]
The logo shown here is not the final one since I made some more minor tweaks in #114. Find the latest logo in our `assets` folder.
